web/models.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
```

Comment:
Go discourages collecting types together into a 'models' package/file.
Instead, it is standard practice to keep type definitions close to the code that uses them.

For example, if a type is only used in one file (I think that is true for many of the types in this file), then the type definition should be in that file. If the type is used throughout the file, then put it is common to put it towards the top. If it is used in only a few places, then put it close to the first use.

If a file is used in many files, then usually there is a public API that kind of owns the type definition -- if so, put it there.

See for example https://rakyll.org/style-packages/

Reply:
makes sense, moved these structs close to the functions which are using them in web.go